echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineConfigsPollingAgent.groovy

```diff
@@ -78,20 +78,20 @@ class PipelineConfigsPollingAgent extends AbstractPollingAgent {
     try {
       log.info("Running the pipeline configs polling agent...")
 
-      /**
-       * Only interested in pipelines that have triggers and that too scheduled triggers
-       */
+      // Only interested in pipelines that have triggers and that too scheduled triggers
       def pipelines = pipelineCache.getPipelines().findAll {
         it.triggers && it.triggers.any { TRIGGER_TYPE.equalsIgnoreCase(it.type) }
       }
 
+      def triggerRepo = new TriggerRepository(pipelines)
```

Comment:
as far as the `PipelineConfigsPollingAgent` is concerned, this is where the magic happens

Reply:
`s/triggerRepo/triggerRepository` for consistentcy